app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { Field, Heading, Modal, t } from "@bloom-housing/ui-components"
+import { useForm } from "react-hook-form"
+import { RailsLotteryBucketsDetails } from "../../api/types/rails/listings/RailsLotteryBucketsDetails"
+import { RailsListing } from "../listings/SharedHelpers"
+import { ListingDetailsLotteryPreferences } from "./ListingDetailsLotteryPreferences"
+
+interface ListingDetailsLotteryModalProps {
+  isOpen: boolean
+  listing: RailsListing
+  lotteryBucketDetails: RailsLotteryBucketsDetails
+  onClose: () => void
+}
+
+export const ListingDetailsLotteryModal = ({
+  isOpen,
+  listing,
+  lotteryBucketDetails,
+  onClose,
+}: ListingDetailsLotteryModalProps) => {
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register } = useForm({ mode: "onChange" })
+
+  return (
+    <Modal onClose={onClose} open={isOpen} title="">
+      <div className="w-96">
+        <header className="pb-4 text-center">
+          <Heading>{t("listings.lottery.lotteryResults")}</Heading>
+          <h2 className="font-sans font-semibold text-sm uppercase">{listing.Name}</h2>
```

Comment:
this could be a `<Heading priority={2}>` right?

Reply:
actually, nix that. I think it makes sense to keep the h2 with utility classes since there isn't a style-equivalent in Bloom (I don't think). if we find we're using this set of styles a lot though we could make a component